Lombiq.Hosting.Tenants.FeaturesGuard/Extensions/OrchardCoreBuilderExtensions.cs

```diff
@@ -0,0 +1,23 @@
+using Lombiq.Hosting.Tenants.FeaturesGuard.Models;
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class OrchardCoreBuilderExtensions
```

Comment:
As I've written [here](https://github.com/Lombiq/Hosting-Tenants/pull/41#discussion_r1012341787):

> Since we'll repeatedly use this, you can also add an extension method similar to AllowMiniProfilerOnAdmin() in Orchard, so one doesn't need to write the whole ceremony, just **a) set EnableFeatureIfOtherFeatureIsEnabled if you have something custom** or b) set the Azure Storage config directly since that'll be a common use-case. (

Reply:
Yes, this is what I intended to do with `ConfigureFeaturesGuardForAzureStorage()` -- so it's possible to set the Azure Storage config with this extension method, or add custom configuration in _appsettings_. Which part of this did I get wrong?

Reply:
In addition to this, add a second extension method to set EnableFeatureIfOtherFeatureIsEnabled if you have something custom, not related to Azure and Media.